src/main/java/com/breskul/bibernate/proxy/interceptor/LazyInterceptor.java

```diff
@@ -24,7 +24,7 @@ public LazyInterceptor(Supplier<T> supplier) {
 
   /**
    * Intercepts a method call on a lazily initialized object and initialize real object when lazy
-   * object first accessed.
+   * object relationType accessed.
```

Comment:
I believe it's mistake after refactoring

Reply:
fixed